cob_cartesian_controller/src/cartesian_controller.cpp

```diff
@@ -460,56 +396,47 @@ cob_cartesian_controller::CartesianActionStruct CartesianController::acceptGoal(
 
     if(action_struct.name == "move_lin")
```

Comment:
definitely!
we should use `uint8` defines as in the cob_cartesian_controller Profile.msg